local-cli/cliEntry.js

```diff
@@ -15,6 +15,7 @@ const defaultConfig = require('./default.config');
 const dependencies = require('./dependencies/dependencies');
 const generate = require('./generate/generate');
 const library = require('./library/library');
+const link = require('./rnpm/link/src/link');
```

Comment:
Could this come from a dev-dependency on `rnpm-plugin-link` instead of being committed directly to react-native?